src/renderer/store/editor.js

```diff
@@ -815,6 +815,12 @@ const actions = {
 
   ASK_FOR_IMAGE_PATH ({ commit }) {
     return ipcRenderer.sendSync('mt::ask-for-image-path')
+  },
+
+  LISTEN_WINDOW_ZOOM () {
+    ipcRenderer.on('mt::window-zoom', (e, zoomFactor) => {
+      webFrame.setZoomFactor(zoomFactor)
```

Comment:
I didn't know why it's has no effect on macOS system when I change the zoom factor?

Reply:
You could try to build a simple example with `electron-api-demos` with Electron 5.x and eventually report this issue, maybe upstream issue?

Reply:
HAHA, I don't need to change zoom factor, I'll test it later.